dockers/cuda-nightly/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG CUDNN_VERSION=7
+ARG CUDA_VERSION=10.1
+
+FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel
+
+ARG PYTHON_VERSION
+ARG PYTORCH_VERSION
+ARG PYTORCH_CHANNEL
+ARG CONDA_VERSION=4.7.12
+
+SHELL ["/bin/bash", "-c"]
+
+ENV HOROVOD_GPU_ALLREDUCE=NCCL
+ENV HOROVOD_GPU_BROADCAST=NCCL
+ENV HOROVOD_WITH_PYTORCH=1
+ENV HOROVOD_WITHOUT_TENSORFLOW=1
+ENV HOROVOD_WITHOUT_MXNET=1
+ENV HOROVOD_WITH_GLOO=1
+ENV HOROVOD_WITHOUT_MPI=1
+ENV PATH="$PATH:/root/.local/bin"
+ENV MAKEFLAGS="-j$(nproc)"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    cmake \
+    git \
+    curl \
+    ca-certificates
+
+COPY ./tests/install_AMP.sh install_AMP.sh
+COPY ./requirements/extra.txt requirements-extra.txt
+COPY ./requirements/test.txt requirements-tests.txt
+COPY ./requirements/examples.txt requirements-examples.txt
+
+# install conda and python
+RUN curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh && \
+    chmod +x ~/miniconda.sh && \
+    ~/miniconda.sh -b -p /root/conda && \
+    # ~/miniconda.sh -b -p /home/containeruser/conda && \
+    rm ~/miniconda.sh && \
+    /root/conda/bin/conda clean -ya && \
+    /root/conda/bin/conda install -y python=$PYTHON_VERSION && \
+    /root/conda/bin/conda install -y pytorch=$PYTORCH_VERSION torchvision cudatoolkit=$CUDA_VERSION -c $PYTORCH_CHANNEL
+
+# add conda to path
+ENV PATH /root/conda/bin:$PATH
+
+# install dependencies
+RUN \
+    # Install AMP
+    bash install_AMP.sh && \
```

Comment:
Do we need to install amp for pytorch >=1.6?

Reply:
yes... because some 1.6 users still like apex

Reply:
alright.